test/powershell/Modules/Microsoft.PowerShell.Utility/ConvertTo-Json.Tests.ps1

```diff
@@ -28,10 +32,8 @@ Describe 'ConvertTo-Json' -tags "CI" {
         })
         $null = $ps.BeginInvoke()
         # wait for verbose message from ConvertTo-Json to ensure cmdlet is processing
-        Wait-UntilTrue { $ps.Streams.Verbose.Count -gt 0 }
-        $null = $ps.BeginStop($null, $null)
-        # wait a bit to ensure state has changed, not using synchronous Stop() to avoid blocking Pester
-        Start-Sleep -Milliseconds 100
+        Wait-UntilTrue { $ps.Streams.Verbose.Count -gt 0 } -IntervalInMilliseconds 50
+        $null = $ps.Stop()
```

Comment:
Per comment on line 33, `Stop()` is synchronous and in the chance it blocks, Pester is blocked.  Perhaps we can change the sleep on line 34 to `Wait-UntilTrue` where `State` changes?

Reply:
I think we could discuss this in #7819 and perhaps remove the tests at all.

Reply:
If you can revert changes to this test in this PR, I think we can move forward with this PR and resolve this test in the other PR.

Reply:
Reverted in the PR and moved to #7819.